hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieClusteringJob.java

```diff
@@ -249,4 +263,10 @@ private int handleErrors(HoodieCommitMetadata metadata, String instantTime) {
     return -1;
   }
 
+  private HoodieTableMetaClient getMetaClient() {
+    if (metaClient == null) {
+      metaClient = HoodieTableMetaClient.builder().setConf(jsc.hadoopConfiguration()).setBasePath(cfg.basePath).setLoadActiveTimelineOnLoad(true).build();
+    }
+    return metaClient;
+  }
```

Comment:
Consider moving this to `UtilHelpers`. We can reuse in `HoodieCompactor` as well.

Reply:
Got it.  Fixed.